src/Avalonia.DesignerSupport/Remote/HtmlTransport/HtmlTransport.cs

```diff
@@ -174,13 +170,6 @@ async void SocketWorker()
             }
         }
         
-        public void Dispose()
-        {
-            _pendingSocket?.Dispose();
-            _simpleServer.Dispose();
```

Comment:
It seems to me that using the Disposible pattern is a controversial decision. The reasons for this can be read [here](https://www.mono-project.com/docs/advanced/pinvoke/#properly-disposing-of-resources), starting with
> Frequently IDisposable examples will provide a virtual Dispose(bool) method

Reply:
Yeah IMO this warning is outdated and we shouldn't be following it.